dss/index/es/validator.py

```diff
@@ -34,6 +37,33 @@ def remove_json_fields(json_data: dict, path: List[str], fields: List[str]) -> N
     for field in fields:
         current.pop(field)
 
+schema_url_regex = re.compile(r'/(?P<major_version>[0-9]+)\.[0-9]+\.[0-9]+[\w\d/]*/(?P<type>[\w\d]+)(.json)?$')
+
+
+class HcaSchema(namedtuple('HcaSchema', ['url', 'version', 'type'])):
+    getters = ['describedBy', 'core']
+
+    @classmethod
+    def from_schema(cls, schema: dict) -> 'HcaSchema':
+        getter = [i for i in schema.keys() if i in cls.getters]
```

Comment:
`[k for k,v in HcaSchema.__dict__.items() if isinstance(v,staticmethod)]`

and don't call it `getter`